app/src/main/java/com/hadiyarajesh/notex/ui/component/AllNotesComponents.kt

```diff
@@ -0,0 +1,89 @@
+package com.hadiyarajesh.notex.ui.component
+
+import androidx.compose.foundation.layout.*
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material3.Card
+import androidx.compose.material3.CardDefaults
+import androidx.compose.material3.Divider
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import com.hadiyarajesh.notex.database.converter.InstantConverter
+import com.hadiyarajesh.notex.database.entity.Note
+import java.time.Instant
+
+@Composable
+fun NoteCard(note: Note) {
+    Card(
+        elevation = CardDefaults.cardElevation(),
+        modifier = Modifier
+            .fillMaxWidth(),
+        shape = RoundedCornerShape(16.dp)
+    ) {
+        Column(
+            verticalArrangement = Arrangement.Center,
+            modifier = Modifier.padding(18.dp)
+        ) {
+            note.title?.let {
+                TextSemiBold(
+                    content = it,
+                    null,
+                    MaterialTheme.typography.titleLarge, Color.Black
+                )
+            }
+            Row(
+                Modifier
+                    .fillMaxWidth()
+                    .padding(top = 16.dp)
+            ) {
+                Row(
+                    modifier = Modifier
+                        .fillMaxWidth()
+                        .height(IntrinsicSize.Min)
+                        .weight(1f)
+                ) {
+                    TextSemiBold(
+                        content = "Succeed",
+                        Modifier.padding(end = 16.dp)
+                    )
+                    Divider(
```

Comment:
Also, we can keep all these types of Composables in a separate folder **`/composable/elements`** and keep the definition based on the properties.

Reply:
@sumanabhi Yes, we already have `/ui/component/` directory for this.